Makefile

```diff
@@ -231,6 +250,7 @@ test\:coverage-html:
 
 .PHONY: test\:changes
 test\:changes:
+	@$(MAKE) deps:lens
```

Comment:
todo: This rule assumes deps are there. If you see `.github/workflows/detect-change.yml`:

Should instead add `make deps:test-all` according to above suggestion and then just change in `detect-change.yml` to: 

```
      - name: Build dependencies
        run: |
          make deps:modules
          make deps:test-all
```

Reply:
But that would hide the dependency from the make file.  With the current setup `make test:changes` is independent and complete - including in documenting its own dependencies.  Why would I want to hide that/force people to find and look through a yaml file?

Reply:
This would build lens deps every time you run `test:changes`. The `Build dependencies` already has deps it needs before calling this `test:changes`, that is because it runs it for the first time.

Reply:
Why would I want to have to make two calls to run the change detector?  I just want one call, and I want it to run the code in front of me, not a stale binary from ages back.  These make actions are very handy outside of the CI as well as within it.